server/README.md

```diff
@@ -86,6 +86,9 @@
 - POST `/teachers`
   - Body: Teacher object
   - Response: Response status
+-DELETE `/teachers/{teacher_id}/lectures/{lecture_id}`
```

Comment:
Maybe `DELETE /lectures/{lecture_id}` could be better, what do you think?

Reply:
Yes,I was about to write it.  It's probably better to follow `DELETE /lectures/{lecture_id}`. We should change the API call inside LecurePage though